include/lsst/sphgeom/Pixelization.h

```diff
@@ -23,6 +23,11 @@
 #ifndef LSST_SPHGEOM_PIXELIZATION_H_
 #define LSST_SPHGEOM_PIXELIZATION_H_
 
+// PyInt_GetMax is not in the python 3 C API, so we define it for python >=3
+#if PY_MAJOR_VERSION >= 3
+#define PyInt_GetMax(x) 2<<31
+#endif
```

Comment:
Building or not is not the issue. Putting Python API definitions into a header file that is not part of a Python interface is the problem. This define should absolutely go in `pixelizations.i` because it's part of the Python interface and that is where the Python API call is being used.

It looks like the code in `pixelizations.i` could work with just 64-bit but I'd like @smonkewitz to comment before we merge as it might be that we should be tweaking the code there and not using `PyInt_GetMax` at all.